cmd/pdcli/oncall/cmd.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright © 2021 Red Hat, Inc
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package oncall
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/constants"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/output"
+	"github.com/openshift/pagerduty-short-circuiter/pkg/pdcli"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "oncall",
+	Short: "oncall to the PagerDuty CLI",
+	Long:  "Running the pdcli oncall command will display the current primary and secondary oncall SRE",
+	Args:  cobra.NoArgs,
+	RunE:  OnCall,
+}
+
+type User struct {
+	EscalationPolicy string
+	OncallRole       string
+	Name             string
+	Start            string
+	End              string
+}
+
+//Oncall implements the fetching of current roles and names of users
+func OnCall(cmd *cobra.Command, args []string) error {
+
+	var call pagerduty.ListOnCallOptions
```

Comment:
`call` doesn't seem like a relevant variable name, can you instead change it to something like `opts` or `callOpts`?

Reply:
+1

Reply:
Since in this case action(ListOnCallOptions) itself suggests what it is going to return, I would go with onCallOptions or pdOnCallOptions something like that